### DISCUSSION TITLE
Which folder to select?

### DISCUSSION THREAD

**Discussion by jjonescz** [ipasimulator/ipasim#10] [Q&A] [answered]
@dananothername Let's discuss your problem here, it's off-topic in #5.

> ![Screenshot_6](https://user-images.githubusercontent.com/101902085/212076368-a535fea8-164f-4c80-88e2-294c71cc6846.png)
> i thing i get it ,what folder should i select 

> _Originally posted by @dananothername in https://github.com/ipasimulator/ipasim/issues/5#issuecomment-1380344073_

**Comment by jjonescz** [comment, marked as answer]
You need to download the [samples ZIP](https://github.com/ipasimulator/ipasim/releases/download/v1.0/ipasim-samples-v1.0.zip), extract it, and select one of the folders there. That's what I was suggesting already in https://github.com/ipasimulator/ipasim/issues/5#issuecomment-1380214179.

**Reply by dananothername** [reply]
![Screenshot_1](https://user-images.githubusercontent.com/101902085/212081524-a795b24e-c213-446f-bca2-ccf41b147fd8.png)
choosed folder
![Screenshot_2](https://user-images.githubusercontent.com/101902085/212081678-e6ea4693-1d5e-4302-a7fd-de9c8663306f.png)
![Screenshot_3](https://user-images.githubusercontent.com/101902085/212082019-b4b4357c-30b2-4a1f-8310-8d6bda67c080.png)
started for a sec then
red unicorne error , the app crush 
that was benchmark folder

**Reply by jjonescz** [reply]
The other samples work for you? Anyway it would help to know what the error says.

**Comment by dananothername** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/101902085/212085226-d794a6b7-6dfe-4691-9d9c-c971f000bff3.png)
hard to tell its stuck like this
i think the issue in my pc ,what r the requirement

**Reply by jjonescz** [reply]
It doesn't really seem stuck. Unless the log shows any error, it works like this. The benchmark app just shows some timings and that's it. From the screenshots it looks like it works fine.

**Reply by dananothername** [reply]
then what should i select to run a ios game
samples show 5+10=15
and hello world show hello world

**Reply by jjonescz** [reply]
You need an `.ipa` file for the iOS game you want to run. But I must disappoint you, no complex game will probably work. See [project status](https://github.com/ipasimulator/ipasim#project-status). That's also why we need more contributors for this project as discussed in https://github.com/ipasimulator/ipasim/issues/5.

**Reply by dananothername** [reply]
![Screenshot_1](https://user-images.githubusercontent.com/101902085/212096102-85b068b6-7345-4f97-b397-e3c0402fe45d.png)
say i cant run this one
ok ill learn c++ in months
i found this discord specified in emulators programming languages
https://discord.gg/BnNswgwtWY
also emulation wiki recommended your app and qemu ,i am having bad time installing it too
